packages/ses/src/lockdown.js

```diff
@@ -300,6 +315,16 @@ export const repairIntrinsics = (options = {}) => {
     optGetStackString,
   );
   globalThis.console = /** @type {Console} */ (consoleRecord.console);
+  // eslint-disable-next-line no-underscore-dangle
+  if (typeof (/** @type {any} */ (globalThis.console)._times) === 'object') {
+    hostIntrinsics.SafeMap = getPrototypeOf(
```

Comment:
This probably needs more explanation.

Reply:
Oh yeah, just getting it to pass first. But basically NodeJS has hidden intrinsics, supposed to be internal only but one of them leaks through an "underscore" property of their console.